Doc/whatsnew/3.10.rst

```diff
@@ -558,6 +558,23 @@ Removed
   the :mod:`collections` module.
   (Contributed by Victor Stinner in :issue:`37324`.)
 
+* The ``loop`` parameter has been removed from most of :mod:`asyncio`\ 's
+  :doc:`high-level API <../library/asyncio-api-index>` following deprecation
+  in Python 3.8.  The motivation behind this change is multifold:
+
+  1. This simplifies the high-level API.
+  2. The functions in the high-level API have been implicitly getting the
+     current thread's running event loop since Python 3.6.  There isn't a need to
```

Comment:
```suggestion
     current thread's running event loop since Python 3.7.  There isn't a need to
```

Minor nit: the change was made after the addition of [`asyncio.get_running_loop()`](https://docs.python.org/3/library/asyncio-eventloop.html?highlight=get_running_loop#asyncio.get_running_loop), which was added in 3.7 rather than 3.6.